driver_example.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
```

Comment:
I would place all these `driver_example/template` files in one/two folders into LabDrivers, maybe adding a README in this folder which explains the difference between the `template.py`, the `*_rw.py` and the `*_rwx.py`.

Reply:
(META-DISCUSSION ABOUT WORKFLOW : this can be done in one PR, typically 1-2 commits, will be easy to review and can be merged quickly)

Reply:
That would most definitely be a good idea! I was planning on referring to these templates/examples in the documentation I've begun writing for PropertiesWidget and FunctionsWidget.

In short, rw is read/write, which encompasses measure, set, and get (PropertiesWidget addition)
rwx is read/write/execute, which encompasses measure, set, get, and run (PropertiesWidget and FunctionsWidget addition)

As of right now, a driver only requires reading abilities. writing/executing are both optional, and one could have just rx or rw, but not wx, x, or w